### PR TITLE
Compute date difference instead of time difference

### PR DESCRIPTION
**Pull request by rjchee**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -86,15 +86,22 @@ func lastWinHandler(w http.ResponseWriter, r *http.Request) {
...
-			now := time.Now()
+			userTime := time.Now().UTC()
+			log.Printf("Current time in UTC: %s", userTime.Format(time.UnixDate))
 			tzs, ok := r.URL.Query()["tz"]
 			if ok && len(tzs[0]) > 0 {
 				tz, err := strconv.Atoi(tzs[0])
-				if err != nil {
-					now = now.Add(-time.Minute * time.Duration(tz))
+				if err == nil {
+					// get the user's time as if it were in UTC, i.e. 10:30pm
```
Comment (christodenny) [inline main.go]: Nit, capitalize get

**Review by williamvwang — APPROVED** [review]

File: main.go [hunk 2]
```diff
@@ -86,15 +86,22 @@ func lastWinHandler(w http.ResponseWriter, r *http.Request) {
 				fmt.Fprintf(w, "Error parsing date %s: %s", date, err)
 				return
 			}
-			now := time.Now()
+			userTime := time.Now().UTC()
```
Comment (christodenny) [inline main.go:91]: Can we add a comment about why we convert this to UTC here?

File: main.go [hunk 3]
```diff
@@ -86,15 +86,22 @@ func lastWinHandler(w http.ResponseWriter, r *http.Request) {
...
+				if err == nil {
+					// get the user's time as if it were in UTC, i.e. 10:30pm
+					// PST would become 10:30pm UTC since the ESPN date is
+					// interpreted as UTC
+					userTime = userTime.Add(-time.Minute * time.Duration(tz))
+					log.Printf("Current time in UTC after translation: %s", userTime.Format(time.UnixDate))
 				}
 			}
-			daysSinceWin := int(now.Sub(t).Hours()) / 24
+			userDate := userTime.Truncate(24 * time.Hour)
```
Comment (christodenny) [inline main.go]: Can get rid of the truncate and just use the int division (divide by 24) to truncate everything smaller than days

File: main.go [hunk 4]
```diff
@@ -86,15 +86,22 @@ func lastWinHandler(w http.ResponseWriter, r *http.Request) {
...
+			log.Printf("Current date in UTC: %s", userDate.Format(time.UnixDate))
+			daysSinceWin := int(userDate.Sub(t).Hours()) / 24
```
Comment (christodenny) [inline main.go]: Just in case set daysSinceWin to max(this formula, 0) in case we somehow get negative?

**Review by christodenny — CHANGES_REQUESTED** [review]

**Review by christodenny — APPROVED** [review]
ship it!

**Pull request merged**